did-onion/src/lib.rs

```diff
@@ -95,7 +96,13 @@ impl DIDResolver for DIDOnion {
             Err(meta) => return (meta, Vec::new(), None),
             Ok(url) => url,
         };
-        let mut client_builder = reqwest::Client::builder();
+
+        let mut headers = reqwest::header::HeaderMap::new();
+        headers.insert(
+            "User-Agent",
+            reqwest::header::HeaderValue::from_static(VERSION),
```

Comment:
User agent string should include name as well as version. e.g.:
`reqwest::header::HeaderValue::from_static(format!("ssi/{}", VERSION)`

More info: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent